### PR TITLE
Add WMT evaluation script

### PR DESCRIPTION
**Pull request by jaketae**
**Note**: This is work in progress.

This PR implements the following:
* Load the English validation portion of WMT 19
* Load an AR model to run CLM, with `stride` tokens given as context
* Report perplexity, normalized by the number of tokens

Estimated runtime on GPU is ~1 minute; CPU, ~10 minutes.

This PR should be merged after TyDiQA is integrated to `main` to reduce conflicts in `simple_benchmark.py`, which should eventually support all three simple evaluation datasets (TyDiQA, WMT, LAMBADA).

### CODE REVIEW

File: evaluation/scripts/wmt.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        drop_last=True,
+    )
+    log_likelihoods = []
+    for input_ids in tqdm(loader):
+        input_ids = input_ids.to(device)
+        target_ids = input_ids.clone()
+        target_ids[:, :-stride] = -100
+        with torch.no_grad():
+            outputs = model(input_ids, labels=target_ids)
+            log_likelihood = outputs[0]
```
Comment (tttyuntian) [inline evaluation/scripts/wmt.py:37]: This is a minor point. I prefer to compute the total log_likelihood of a batch to improve code readability. Need to compute a `target_length` before this line.

```suggestion
            log_likelihood = outputs[0] * target_length
```

File: evaluation/scripts/wmt.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        log_likelihoods.append(log_likelihood)
+    perplexity = torch.exp(torch.stack(log_likelihoods).sum() / len(loader))
```
Comment (tttyuntian) [inline evaluation/scripts/wmt.py:39]: If the above suggestion is taken, then this line needs a tweak. `num_targets` is the total number of target words.

```suggestion
    perplexity = torch.exp(torch.stack(log_likelihoods).sum() / num_targets)
```

File: evaluation/scripts/wmt.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+from tqdm import tqdm
+from transformers import AutoModelForCausalLM, AutoTokenizer
+
+from ..datasets.wmt import WMTEnglishDataset
+
+
+def main(args):
+    """adapted from https://huggingface.co/transformers/perplexity.html"""
+    stride = args.stride
+    device = torch.device(args.device)
```
Comment (tttyuntian) [inline evaluation/scripts/wmt.py:14]: We currently use this in simple benchmark script. 

```suggestion
    device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

**Review by tttyuntian — COMMENTED** [review]
Thanks for the work, and it's clean and concise! I dropped some comments for you :)

**Comment by jaketae** [comment]
Superseded by #58.
